src/containers/login/index.js

```diff
@@ -7,9 +7,36 @@ import './styles.css';
 const Login = () => (
     <div className="login">
         <div className="container">
-            Container
+
+                <br></br>
+ 
+               <div className="container-2">
+
+
+              
+               <div className="cont-email">
+               <input className="e-mail" type="text" placeholder= "     E-mail" ></input>
+               </div>
+               <br></br>
+              
+              
+              <div className="cont-password"> 
+              <input className="password" type="password" placeholder= "    Password"></input>
+              </div>
+                
+               <hr className="hr-one"></hr> 
```

Comment:
How many hr's do you have? Not need to number this. Also, use a more descriptive class as for example `separator-line`